### DISCUSSION TITLE
Welcome to go-guerrilla Discussions!

### DISCUSSION THREAD

**Discussion by phires** [phires/go-guerrilla#48] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by pkarc** [comment]
Was thinking on opening a discord channel, what do you guys think?

**Reply by pkarc** [reply]
@phires if you can spend some time I would love to build a roadmap and bring all the @flashmob ideas and branches to the table, what do you think? lets meet!

**Reply by phires** [reply]
Hi, sorry, I am a bit swamped currently, so I do not have that much time for this project...
I am not to fond of Discord, but it might help to better bounce some ideas around. We can absolutly talk about building a new roadmap and a future plan for this project, so it gets some tracktion again.
This year will be tight for me, but let's see that we can connect in January?

**Reply by pkarc** [reply]
Sure, January sounds good for me, just ping me when you are available again.
